src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -128,6 +124,9 @@ class AccessibleView extends Disposable {
 			this._keyListener = this._register(this._editorWidget.onKeyUp((e) => {
 				if (e.keyCode === KeyCode.Escape) {
 					this._contextViewService.hideContextView();
+					// Delay to allow the context view to hide #186514
+					setTimeout(() => provider.onClose(), 100);
```

Comment:
It's less hacky if `setTimeout(() => provider.onClose())` works as that will just run it on the next frame